evm_arithmetization/src/cpu/kernel/tests/log.rs

```diff
@@ -201,6 +201,6 @@ fn test_log_5() -> Result<()> {
     interpreter.set_global_metadata_field(GlobalMetadata::LogsLen, 0.into());
     interpreter.set_global_metadata_field(GlobalMetadata::LogsDataLen, 0.into());
 
-    assert!(interpreter.run().is_err());
+    assert!(interpreter.run(None).is_err());
```

Comment:
There are two such assertions in the code, but I'm not sure fixing these would be in the scope of this (already pretty big) PR (especially since with your suggested changes, this file would not be touched at all otherwise). I can still try to fix these if you think we should still do it here though!

Reply:
Let's do it in another PR!

Reply:
@LindaGuiga could you add a TODO for it so we don't forget? Thanks!